lms/djangoapps/grades/new/course_data.py

```diff
@@ -0,0 +1,89 @@
+from lms.djangoapps.course_blocks.api import get_course_blocks
+from xmodule.modulestore.django import modulestore
+from ..transformer import GradesTransformer
+
+
+class CourseData(object):
+    """
+    Houses course data and intelligently gets and caches the
```

Comment:
Not needed for the `CourseData` class - since it's an ephemeral, request-specific, in-memory class only.  (Worth adding a comment if that caused confusion.)